### PR TITLE
feat: toast notifications ui

### PR DESCRIPTION
**Pull request by ouichniaziz**
## Description
Apply UI design to Toast notifications.

## Related Issue
https://github.com/typehero/typehero/issues/943

## Screenshots/Video (if applicable):

https://github.com/typehero/typehero/assets/80454936/663a56da-cf67-49ef-91a7-da1458d46ec5

Screenshot 2023-12-09 at 10 36 17 AM
Screenshot 2023-12-09 at 11 10 05 AM
Screenshot 2023-12-09 at 11 11 55 AM

### CODE REVIEW

**Comment by bautistaaa** [comment]
looks good. one thing thats missing is border on the "Try Again" button in light mode. Theres a border in dark mode. Example: submitting a challenge to see

CleanShot 2023-12-09 at 08 55 10@2x

**Comment by ouichniaziz** [comment]
fixed: 
Screenshot 2023-12-09 at 6 13 49 PM

File: apps/web/src/app/[locale]/settings/_components/edit-profile-tab.tsx [hunk 1]
```diff
@@ -66,7 +66,8 @@ function ProfileForm({ user }: Props) {
     try {
       await updateProfile(data);
       toast({
-        title: 'profile updated',
+        title: 'Profile Updated',
+        description: 'You profile has been successfully updated.',
```
Comment (bautistaaa) [inline apps/web/src/app/[locale]/settings/_components/edit-profile-tab.tsx:70]:
```suggestion
        description: 'Your profile has been successfully updated.',
```

**Comment by bautistaaa** [comment]
one last thing, is its missing the inset shadow from the original mocks so that it adds a bit more depth to the circles

**Comment by bautistaaa** [comment]
i can or someone else can fast follow cause this is good enough ™

**Comment by ouichniaziz** [comment]
Screenshot 2023-12-10 at 12 59 17 AM

**Review by bautistaaa — APPROVED** [review]

**Pull request merged**